Update CDN Documentation (#1110)
zeit/docs: pages/docs/v2/network/compression.mdx

import Doc from '~/components/layout/docs'
import Caption from '~/components/text/caption'
import { InlineCode } from '~/components/text/code'
import { GenericLink } from '~/components/text/link'

export const meta = {
  title: 'Compression',
  description:
    'How ZEIT Now serves content with compression and which methods are used.',
  editUrl: 'pages/docs/v2/network/compression.mdx',
  lastEdited: '2019-08-12T19:45:54.000Z'
}

The ZEIT Network layer implements two compression methods; [Gzip](https://en.wikipedia.org/wiki/LZ77_and_LZ78#LZ77) and [Brotli](https://en.wikipedia.org/wiki/Brotli). We highly recommend leveraging ZEIT Network compression over a self hosted solution.

> Using the provided compression methods will provide you with the best performance and you do not have to worry about maintaining the code responsible for compressing your responses.

Reviewer: ```suggestion
Using the provided compression methods will result in the best performance and least maintenance.
```

> Any program that requests resources from your deployment needs to define the [Accept-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding) header to opt into compression. Many clients (browsers like Chrome, Firefox and Safari) already do this out of the box. If you're making a request from another type of client make sure you use the `Accept-Encoding` header to take advantage of compression.

Reviewer 2: ```suggestion
Any client that makes requests to your deployment needs to define the [Accept-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding) header to opt into compression. Many clients (browsers like Chrome, Firefox and Safari) already do this out of the box. If you're making a request from another type of client make sure you use the `Accept-Encoding` header to take advantage of compression.
```

> Any program that requests resources from any of your deployments needs to define the [Accept-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding) header in order for the compression mechanisms to kick in. Many clients (browsers like Chrome, Firefox and Safari) already do this out of the box. If you're making a request from another type of client make sure you use the `Accept-Encoding` header to take advantage of compression.

Reviewer: ```suggestion
Any program that requests resources from your deployment needs to define the [Accept-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding) header to opt into compression. Many clients (browsers like Chrome, Firefox and Safari) already do this out of the box. If you're making a request from another type of client make sure you use the `Accept-Encoding` header to take advantage of compression.
```
